config/custom-environment-variables.json

```diff
@@ -60,23 +60,12 @@
           "__name": "NEUTRINO_BITCOIN_MAINNET",
           "__format": "json"
         }
-      },
-      "litecoin": {
-        "testnet": {
-          "__name": "NEUTRINO_LITECOIN_TESTNET",
-          "__format": "json"
-        },
-        "mainnet": {
-          "__name": "NEUTRINO_LITECOIN_MAINNET",
-          "__format": "json"
-        }
       }
     }
   },
 
   "units": {
     "bitcoin": "UNITS_BITCOIN",
```

Comment:
This is invalid json due to the trailing comma and as a result the app will not start with this change.

Reply:
Ithink I fixed it

Reply:
Thank you, I think I fixed it.